CMakeLists.txt

```diff
@@ -1,7 +1,10 @@
 cmake_minimum_required(VERSION 3.11)
 
-include(CMakeDependentOption)
-cmake_dependent_option(RGM_ENABLE_GRPC_SERVER "Enable the GRPC client plugin for compilation and code analysis." ON "WIN32" OFF)
+if (WIN32)
+  option(RGM_ENABLE_GRPC_SERVER "Enable the GRPC client plugin for compilation and code analysis." ON)
+else()
+  option(RGM_ENABLE_GRPC_SERVER "Enable the GRPC client plugin for compilation and code analysis." OFF)
+endif()
```

Comment:
Why did you do this?

Reply:
cause your way completely hides the option on linux mine just disables it

Reply:
also, cause i dont have OCD like u

Reply:
Well it's not really OCD, it's just like if you have to type that string multiple times you are more likely to make a typo and cause a run time bug. Whereas if you make a typo in my way, even though it's more characters, it's instead more likely to be a compile time error.

Reply:
sounds like ocd to me. if you tested that works ill append the pr tho